src/main/cpp/AutonomousModes/RightLeftAuton.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2017-2020 FRC Team 3512. All Rights Reserved.
+
+//=============================================================================
+// File Name: AutonMotionProfile.cpp
+// Description: Drives set distance with motion profiles
+// Author: FRC Team 3512, Spartatroniks
+//=============================================================================
+
+#include <frc2/Timer.h>
+#include <units/units.h>
+
+#include "Robot.hpp"
+
+bool Robot::checkReflectiveStrips() { return true; }
+
+enum class State { kInit, kMoveForward, kWheelManuel, kIdle };
+static State m_state;
+
+void Robot::RightLeftAutonInit() { m_state = State::kInit; }
+
+void Robot::RightLeftAuton() {
+    switch (m_state) {
+        case State::kInit: {
+            frc2::Timer* timer;
+            timer->Reset();
+            robotDrive->resetEncoders();
+            timer->Start();
```

Comment:
This code will crash. You declared a pointer, which is initialized with a random address, then tried to call a function on a Timer object at that random address (which will be gibberish).

You should make a static `frc::Timer` (not `frc2::Timer*`) like the State above, and use that for autonomous mode timing.